### DISCUSSION TITLE
Ticket submitted via /submit route returns error

### DISCUSSION THREAD

**Discussion by sudo-sein** [Peppermint-Lab/peppermint#351] [Q&A]
I used /submit route to test the anonymous submitting functionality, and I tried to access the ticket as admin in dashboard I got following error:
"initialContent must be a non-empty array of blocks, received: This is ticket content"

Here's screenshot of details page.
![image](https://github.com/Peppermint-Lab/peppermint/assets/419502/5e3d319d-6195-4872-9b83-a976d4db168f)

Here's log console for that ticket.
![image](https://github.com/Peppermint-Lab/peppermint/assets/419502/91f96c97-146b-456e-8030-93b5443f37a4)

Seems to me like new submissions are not escaped correctly.

**Comment by hiteshdaswani** [comment]
Same error, i thought it was only me, i even reinstalled my debian and peppermint twice.
i tried with the default user as well as a new admin, i keep getting application error, client side

**Comment by hiteshdaswani** [comment]
Did you get any response @sudo-sein ?

Im stumped i'm looking for a crm or ticketing solution open source, peppermint looked great but the support is kinda lacking.

**Reply by hiteshdaswani** [reply]
hey @sudo-sein 
I joined their discord forum and i found that more people have the error 
https://discord.com/channels/788168309752594443/812305842899779655

check the channel. 

It says 

Using the test tag, solved the issue, version 0.4.3 works
i guess the latest is broken

**Comment by potts99** [comment]
this is now fixed all, apologies
